js/src/hooks/useSsl.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import useAppSelectDispatch from './useAppSelectDispatch';
+
+/**
+ * Call `useAppSelectDispatch` with `"getIsSsl"`.
+ *
+ * @return {useAppSelectDispatch} Result of useAppSelectDispatch.
+ */
+/**
+ * Get store ssl from API. Returns `{ hasFinishedResolution, data, invalidateResolution }`.
+ *
+ * `data` is an object of store ssl mapping. e.g.:
+ *
+ * ```json
+ * {
+ * 		"is Ssl" : "true"
+ * }
+ * ```
+ */
+const useSsl = () => {
```

Comment:
I think we could update the file and the hook name to `useIsStoreSsl` so it matches with `IsStoreSSl`